transit_odp/validate/antivirus.py

```diff
@@ -96,23 +96,24 @@ def scan(self, file_: BinaryIO):
 
         if result.status == "ERROR":
             logger.info("Antivirus scan: FAILED")
-            raise AntiVirusError(file_.name)
+            raise AntiVirusError(file_.name, result.reason)
```

Comment:
For AntiVirusError construct second parameter is line number, 
3rd parameter is message, 
Here we are passing message in line param

Reply:
Fixed this @amitkpmguk